schemas/v1/schema.json

```diff
@@ -0,0 +1,116 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://github.com/dotnet/docfx/schemas/v1/schema.json#",
+    "title": "DocFX document schema meta-schema",
```

Comment:
?

---
In reply to: [126855131](https://github.com/dotnet/docfx/pull/1854#discussion_r126855131) [](ancestors = 126855131)